include/zephyr/net/net_time.h

```diff
@@ -30,14 +30,7 @@ extern "C" {
 
```

Comment:
This is not really an openthread change so commit message could just be `net: Change net_time_t defined type`

Reply:
fixed.